test/Edge-SDV/TestDocs/README.md

```diff
@@ -0,0 +1,4 @@
+# Steps to Compile & Execution of SDV automation suit 
+
+1.  go to edge_core/ folder
```

Comment:
Can we use shell scripts to describe the following actions? e.g. 

```
cd /edge_core && make intergration_test
```

Reply:
resolved